plenum/test/freshness/test_freshness_batch_updates_last_ordered.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+
+from plenum.common.messages.node_messages import PrePrepare
+from plenum.server.replica import Replica
+
+from plenum.common.constants import DOMAIN_LEDGER_ID
+from plenum.common.exceptions import PoolLedgerTimeoutException
+from plenum.server.node import Node
+from plenum.test.helper import freshness, sdk_send_random_and_check, primary_disconnection_time
+from plenum.test.spy_helpers import getSpecificDiscardedMsg
+from plenum.test.view_change.helper import restart_node
+
+# 20 secs, so node could restart
+FRESHNESS_TIMEOUT = 20
+
+
+@pytest.fixture(scope="module")
+def tconf(tconf):
+    with freshness(tconf, enabled=True, timeout=FRESHNESS_TIMEOUT), primary_disconnection_time(tconf, 60):
```

Comment:
I think we can make `primary_disconnection_time` even more (1000), since we don't want it ever happen in this test.

Reply:
ok